mmv1/third_party/terraform/services/secretmanager/resource_secret_manager_secret_test.go.erb

```diff
@@ -426,3 +484,150 @@ resource "google_secret_manager_secret_version" "secret-version-4" {
 }
 `, context)
 }
+
+func testAccSecretMangerSecret_userManagedCmekBasic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+data "google_project" "project" {
+  project_id = "%{pid}"
+}
+resource "google_kms_crypto_key_iam_member" "kms-central-binding-1" {
+  crypto_key_id = "%{kms_key_name_central}"
+  role          = "roles/cloudkms.cryptoKeyEncrypterDecrypter"
+  member        = "serviceAccount:service-${data.google_project.project.number}@gcp-sa-secretmanager.iam.gserviceaccount.com"
+}
+resource "google_kms_crypto_key_iam_member" "kms-central-binding-2" {
+  crypto_key_id = "%{kms_key_name_central_other}"
+  role          = "roles/cloudkms.cryptoKeyEncrypterDecrypter"
+  member        = "serviceAccount:service-${data.google_project.project.number}@gcp-sa-secretmanager.iam.gserviceaccount.com"
+}
+resource "google_kms_crypto_key_iam_member" "kms-east-binding" {
+  crypto_key_id = "%{kms_key_name_east}"
+  role          = "roles/cloudkms.cryptoKeyEncrypterDecrypter"
+  member        = "serviceAccount:service-${data.google_project.project.number}@gcp-sa-secretmanager.iam.gserviceaccount.com"
+}
+resource "google_secret_manager_secret" "secret-basic" {
+  secret_id = "tf-test-secret-%{random_suffix}"
+  
+  labels = {
+    label = "my-label"
+  }
+  replication {
+    user_managed {
+      replicas {
+        location = "us-central1"
+      }
+    replicas {
```

Comment:
```suggestion
      replicas {
```
nit-picks

Reply:
Thanks for the review! I've added the extra whitespaces.